src/Altinn.Common/Altinn.EFormidlingClient/Altinn.EFormidlingClient/Altinn.EFormidlingClient/Altinn.EFormidlingClient/Altinn.EFormidlingClient.xml

```diff
@@ -0,0 +1,1115 @@
+<?xml version="1.0"?>
+<doc>
```

Comment:
I don't think this needs to be checked in.. Do you use it for anything explicitly? I would expect to see the xml in the bin folder, but not at this level